### PR TITLE
Added ability to visit a user's profile

### PR DESCRIPTION
**Pull request by GreenRaccoon23**
Added the ability to view a user's profile after watching his story. In the text which says "made with <3 by <username>", the username is a link to that profile page. The profile page just reuses the splash page, so not much code was added, even though it is across many different files.

### CODE REVIEW

File: client/singleStory/singleStory.html [hunk 1]
```diff
@@ -6,14 +6,17 @@
         <div class="row">
             <div class="act1 col-md-4 player-video" ng-class="act1divclass">
                 <div id="player1" class="center-it"></div>
+                <div id="player1Audio" style="display: none"></div>
```
Comment (jd009) [inline client/singleStory/singleStory.html]: What is this?  It doesn't appear to be related to viewing a user's profile.

Comment (GreenRaccoon23) [inline client/singleStory/singleStory.html]: Oops I forgot to take that out

File: client/splash/splash.js [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
-      story.frames.forEach(function(act) {
-        act.isTextToSpeech = false;
-
-        var previewUrlSrc = act.previewUrl.substring(0, 12)
-        var previewUrlExt = act.previewUrl.substring(act.previewUrl.length -4)
-
-        if (previewUrlSrc === '/api/images/' && previewUrlExt === ".svg") {
-          act.isTextToSpeech = true;
-        }
+.controller('splashCtrl', function ($rootScope, $scope, $state, $window, StoryStorage, $stateParams) {
```
Comment (jd009) [inline client/splash/splash.js]: There's a dependency on dollar sign window but it doesn't look like it's being used anymore.
